wandb/yolov8/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+Compatibility yolov8 module
```

Comment:
no compatibility needed... this doesnt exist yet, we should just have it available at wandb.integrations.yolov8

Reply:
:+1: Removed the compatibility module and changed the usage as per the comment.